packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines -- barely exceeds limit */
```

Comment:
This comment is likely to become incorrect over time (When the file extends, people won't update this line). Can't think of a better one though, so maybe fine?

Reply:
You could move each of the two processing functions to their own files to avoid the error.